### PR TITLE
test(e2e): rename v2 as evm tests and rename old evm tests as legacy

### PR DESCRIPTION
**Pull request by lumtis**
# Description

- Rename v2 as normal EVM tests
- Move old EVM tests in legacy directory, still defined here since it is still supported but will make it easier in the future to remove
- Remove some unused and useless tests (zrc20 swap is already covered in crosschain swap for example)
- Remove V2 migration tests

### CODE REVIEW

**Review by ws4charlie — APPROVED** [review]
looks a nice move to fully switching to v2

File: cmd/zetae2e/local/legacy.go
Comment (swift1337) [inline cmd/zetae2e/local/legacy.go:1]: Maybe `cmd/zetae2e/local/evm_legacy.go`?

**Review by swift1337 — APPROVED** [review]

**Review by gartnera — APPROVED** [review]
CI stamp

File: cmd/zetae2e/local/legacy.go
Comment (lumtis) [reply cmd/zetae2e/local/legacy.go:1]: I think it's fine, just important to mention legacy to know we need to remove these in the future

**Pull request merged**